### PR TITLE
Code improvements as per the current requirements

### PR DESCRIPTION
**Pull request by nieldeokar**
There is new contract 'numberOfOpenSites()' as of March 15 2020, implementation of same is added in above method.

### CODE REVIEW

File: percolation/Percolation.java [hunk 1]
```diff
@@ -88,6 +90,11 @@ private void tryUnion(int rowA, int colA, int rowB, int colB) {
             normalQU.union(indexOf(rowA, colA), indexOf(rowB, colB));
         }
     }
+    
+    // returns the number of open sites
+    public int numberOfOpenSites(){
+            return openCount;
```
Comment (mincong-h) [inline percolation/Percolation.java]:
```suggestion
        return openCount;
```

File: percolation/Percolation.java [hunk 2]
```diff
@@ -88,6 +90,11 @@ private void tryUnion(int rowA, int colA, int rowB, int colB) {
             normalQU.union(indexOf(rowA, colA), indexOf(rowB, colB));
         }
     }
+    
+    // returns the number of open sites
```
Comment (mincong-h) [inline percolation/Percolation.java]:
```suggestion
    /**
     * Returns the number of open sites
     */
```

**Review by mincong-h — COMMENTED** [review]
A few style changes, please. Could you also add your name in the Javadoc?

**Review by nieldeokar — COMMENTED** [review]
added author name

**Review by mincong-h — APPROVED** [review]
Thanks!

**Pull request merged**